t/lib/OpenQA/Test/Case.pm

```diff
@@ -80,7 +80,7 @@ sub find_most_recent_event {
     my ($schema, $event) = @_;
 
     my $results
-      = $schema->resultset('AuditEvents')->search({event => $event}, {limit => 1, order_by => {-desc => 'id'}});
+      = $schema->resultset('AuditEvents')->search({event => $event}, {order_by => {-desc => 'id'}})->slice(0, 1);
```

Comment:
`rows => 1` seems simpler

Reply:
Actually, if we want only one result, we should use `rows => 1` and `->single`, or simply `->find`
See the note about single here: https://metacpan.org/pod/DBIx::Class::ResultSet#single

Reply:
![image](https://user-images.githubusercontent.com/1204189/91448468-3325f080-e87a-11ea-9a6f-0fbcf5c69819.png)

Reply:
But yeah, `find` with `rows=>1` seems to work, and I can drop the `->next`.